spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/adapter/SpringDynamoDBAdapterClient.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
```

Comment:
It's far away 2020. So, Copyright has to be an actual.
I would prefer just 2020, but it depends on Checkstyle we rely here from Spring Cloud.
So, probably `2020-2020`.

Reply:
Copy and paste, fixed. It needs to be `2020-2020`.